packages/block-editor/src/components/block-draggable/index.js

```diff
@@ -75,7 +74,6 @@ const BlockDraggable = ( {
 	return (
 		<Draggable
 			cloneClassname={ cloneClassname }
-			elementId={ elementId }
```

Comment:
The `elementId` prop seems to be dead code, `Draggable` ignores it when `__experimentalDragComponent` is defined, and `BlockDraggable` has that prop hard-coded.